api/nodes/serializers.py

```diff
@@ -212,6 +206,7 @@ def profile_image_url(self, user):
 
     class Meta:
         type_ = 'contributors'
+        target_type_ = 'users'
```

Comment:
So, from a design perspective:

What is the advantage of specifying highly field-specific behavior outside of that field? (in the Meta class)